presto-main/src/test/java/io/prestosql/sql/query/TestColumnMask.java

```diff
@@ -455,10 +455,22 @@ public void testShowStats()
                 USER,
                 new ViewExpression(USER, Optional.of(CATALOG), Optional.of("tiny"), "7"));
 
-        assertThatThrownBy(() -> assertions.query("SHOW STATS FOR (SELECT * FROM orders)"))
-                .hasMessageMatching("\\QSHOW STATS for table with column masking is not supported: orderkey");
-        assertThatThrownBy(() -> assertions.query("SHOW STATS FOR (SELECT orderkey FROM orders)"))
-                .hasMessageMatching("\\QSHOW STATS for table with column masking is not supported: orderkey");
+        assertThat(assertions.query("SHOW STATS FOR (SELECT * FROM orders)"))
+                .matches("VALUES " +
+                        "(cast('orderkey' AS varchar), cast(null AS double), cast(1 AS double), cast(0 AS double), cast(null AS double), cast('7' AS varchar), cast('7' AS varchar))," +
```

Comment:
It would improve readability if you extracted this to   method
```
String statsRow(String columnName, double datasize,....) 
```

also, 

- `cast(1 AS double)` can be expressed shortly with `1e0`
- uppercase `CAST` and `NULL`

Reply:
I simplified it differently. Case fixed.